app/models/user.rb

```diff
@@ -4,7 +4,9 @@ class User < ActiveRecord::Base
          :recoverable,
          :rememberable,
          :trackable
+
   validates :name, presence: true
+  paginates_per 25
```

Comment:
Please don't do this :smile:

I can explain in more depth on Monday, but by doing this you're marrying the model to Kaminari (paging) which should really be a UI thing.